.github/workflows/ci.yml

```diff
@@ -0,0 +1,51 @@
+name: CI
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+jobs:
+  lint:
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install clang-format
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y clang-format
+    - name: Run C++ linter
+      run: |
+        find . -name ‘*.h’ -or -name ‘*.hpp’ -or -name ‘*.cpp’ | xargs clang-format -i --style=file
+        git diff --exit-code
+  build_and_test:
```

Comment:
testは現在未実施だと思います。
あとで追加する予定なのであらかじめこの名称にしたという理解です。

I believe that TEST is currently not implemented. It is my understanding that this name was chosen in advance because it will be added later.

Reply:
@kokosabu 
確認ありがとうございます。曖昧なのは良くないので`build`に変更しました！！

Thanks for your reviewing. I changed its name to `build`. Because It is not good that there are unnecessary ambiguity!!